File: API/models/Airhumidity.js
const mongoose = require('mongoose');

const AirhumiditySchema = mongoose.Schema({
    humidity: {
        type: String,
        required: true
    },
    date: {
        type: Date,
        required: true,
        default: Date.now
    },
    plantId: {
        type: String,
        required: true
    }
});

module.exports = mongoose.model('Airhumidity', AirhumiditySchema, 'airhumidities');

File: App/PlantApp/PlantApp/Model/AirHumidity.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace PlantApp.Model
{
    class AirHumidity
    {
        private int humId;
        private string humidity;
        private DateTime date;

        public int HumId { get => humId; set => humId = value; }
        public string Humidity { get => humidity; set => humidity = value; }
        public DateTime Date { get => date; set => date = value; }
    }
}
